### ISSUE TITLE
Hover build fails with linux

### ISSUE THREAD

**Issue by TT527** [go-flutter-desktop/go-flutter#290]
when i run "hover build linux" command

`hover: Downloading engine for platform linux-x64 at version b863200c37...`
`hover: Failed to download engine: failed to get Content-Length header: strconv.Atoi: parsing "": invalid syntax`

**Comment by Drakirus** [comment]
Does this error occurs on `flutter channel beta` ?

**Comment by TT527** [comment]
> Does this error occurs on `flutter channel beta` ?

yes,i execute this ‘flutter channel beta’,then build successful,but i run it command
`creating glfw window: VersionUnavailable: GLX: Forward compatibility requested but GLX_ARB_create_context_profile is unavailable`

**Comment by Drakirus** [comment]
The `glfw` error might comes from your Drivers.
here are some more information:
https://github.com/go-flutter-desktop/go-flutter/issues/272#issuecomment-535403717
https://github.com/go-flutter-desktop/go-flutter/issues/248#issuecomment-535428357

TL;DR: `hover run --opengl=none`.
